lib/teleterm/daemon/config.go

```diff
@@ -33,8 +34,14 @@ type Config struct {
 	Log              *logrus.Entry
 	GatewayCreator   GatewayCreator
 	TCPPortAllocator gateway.TCPPortAllocator
+	// CreateTshdEventsClientCredsFunc lazily creates creds for the tshd events server ran by the
+	// Electron app. This is to ensure that the server public key is written to the disk under the
+	// expected location by the time we get around to creating the client.
+	CreateTshdEventsClientCredsFunc CreateTshdEventsClientCredsFunc
```

Comment:
nit: Is it a convention to add the `Func` suffix to a field that is a function? (I mean the field name)

Reply:
I've seen it being used in the project.

```
$ rg "Func\s+func" | wc -l
      90
```